src/legacy/ui/ui_nav_links/ui_nav_link.js

```diff
@@ -25,6 +25,7 @@ export class UiNavLink {
       order = 0,
       url,
       subUrlBase,
+      disableSubUrlTracking,
```

Comment:
You'll also need to add this to `src/legacy/ui/ui_apps/ui_app.js` which is also a source of navLinks